### PR TITLE
DM-7693: Package cleanups

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: python/lsst/ctrl/orca/CondorWorkflowConfigurator.py [hunk 1]
```diff
@@ -245,16 +245,16 @@ def _configureSpecialized(self, provSetup, wfConfig):
...
             log.debug("CondorWorkflowConfigurator:configure: about to make logs")
             logDirName = os.path.join(self.localStagingDir, "logs")
             log.debug("CondorWorkflowConfigurator:configure: logDirName = "+logDirName)
             logDirName = os.path.join(self.localStagingDir, "logs")
             os.makedirs(logDirName)
             for visit in visitSet:
-                dirName = os.path.join(logDirName, str(visit))
-                log.debug("making dir "+dirName)
+                dirName = os.path.join(logDirName, visit)
+                log.debug("making dir " + dirName)
```
Comment (hsinfang) [inline python/lsst/ctrl/orca/CondorWorkflowConfigurator.py]: `log.debug("making dir %s", dirName)`?

File: python/lsst/ctrl/orca/CondorWorkflowMonitor.py [hunk 2]
```diff
@@ -173,7 +173,7 @@ def run(self):
                         print("logger handled... and... done!")
                         return
 
-                if (event is not None) or (logEvent is not None):
+                if not event or not logEvent:
```
Comment (hsinfang) [inline python/lsst/ctrl/orca/CondorWorkflowMonitor.py]: I thought our Developer Guide does want us to compare with `None` explicitly? (https://developer.lsst.io/coding/python_style_guide.html#is-and-is-not-should-be-used-when-comparing-to-none)

**Review by hsinfang — APPROVED** [review]
looks fine to me

File: python/lsst/ctrl/orca/CondorWorkflowMonitor.py (hunk 2 above)
Comment (srp3rd) [reply python/lsst/ctrl/orca/CondorWorkflowMonitor.py]: You're absolutely correct.  Fixed here, and in another file.

File: python/lsst/ctrl/orca/CondorWorkflowConfigurator.py (hunk 1 above)
Comment (srp3rd) [reply python/lsst/ctrl/orca/CondorWorkflowConfigurator.py]: I'll fix this;  this also occurred several other places, which I"ll also fix.

**Pull request merged**